src/pcapi/models/offer_type.py

```diff
@@ -331,3 +339,10 @@ class Category(enum.Enum):
     PRESSE = ["Presse en ligne"]
     SPECTACLE = ["Spectacle", "Abonnement spectacles"]
     VISITE = ["Musée, arts visuels et patrimoine"]
+
+
+CATEGORIES_LABEL_DICT = {}
+
+for category in list(Category):
+    for label in category.value:
+        CATEGORIES_LABEL_DICT[label] = category.name
```

Comment:
micro-détail (voire question de goût, peut-être :) ) :

```suggestion
CATEGORIES_LABEL_DICT = {
    label: category.name
    for category in list(Category)
    for label in category.value
}
```

Reply:
aaah merci je préfère largement ça aussi :)